src/challenges.js

```diff
@@ -1,21 +1,45 @@
 /* ======================== CallBacks Practice ============================ */
 const each = (elements, cb) => {
+  for (let i = 0; i < elements.length; i++) {
+    cb(elements[i], i);
+  }
   // Iterates over a list of elements, yielding each in turn to the `cb` function.
   // This only needs to work with arrays.
 };
 
 const map = (elements, cb) => {
+  const result = [];
```

Comment:
Great job!  If you have spare time or come back to this see if you can use the `each` function you created above to pass the tests as well.